@here/harp-mapview/lib/TileObjectsRenderer.ts

```diff
@@ -157,7 +156,9 @@ export class TileObjectRenderer {
             return stableSort(a, b);
         };
 
-        this.m_renderer.setOpaqueSort(painterSortStable);
+        // Temporary workaround due to incorrect comparator type definition:
```

Comment:
Please create a bug on https://github.com/three-types/three-ts-types and mention it here.

Reply:
Done.
https://github.com/three-types/three-ts-types/issues/41